lib/foods/foods.js

```diff
@@ -0,0 +1,64 @@
+$(document).ready(function() {
+  $(loadFoods)
```

Comment:
is there a reason you're wrapping this in a jquery object? why not just execute the function?

Reply:
Good call. I just changed it